tests/qir_test_files/RUSLoopXX-1.ll

```diff
@@ -0,0 +1,131 @@
+; ModuleID = 'RUSLoopXX-1.bc'
```

Comment:
How have you generated this files?

Reply:
(Same question for the BC files)

Reply:
No. This one is an example that has been provided to me by Pete.